### PR TITLE
Hw6 jdi

### PR DESCRIPTION
**Pull request by MariaZuevaMouse**

### CODE REVIEW

File: src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.jdi.tools.DataClass;
+import com.marizueva.laboratory.hw6.entities.MetalAndColorData;
+
+public class MetalAndColorForm extends DataClass<MetalAndColorForm> {
```
Comment (shoggy) [inline src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java]: This class should extend Form with DataClass
For an example check LoginForm

File: src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            expand = ".caret")
+    public Dropdown vegetablesDropdown;
+
+    @FindBy(css = ".dropdown.salad input")
+    public Checklist vegetableList;
+
+    @FindBy(id = "submit-button")
+    public Button submitButton;
+
+    public void fillInWithData(MetalAndColorData data) {
```
Comment (shoggy) [inline src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java]: Would be better to override some existing method

**Review by shoggy — CHANGES_REQUESTED** [review]
Please do not push commented code to the repository

File: src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java (hunk 1 above)
Comment (MariaZuevaMouse) [reply src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java]: fixed

File: src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java (hunk 2 above)
Comment (MariaZuevaMouse) [reply src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java]: fixed

File: src/test/resources/test.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+driver=chrome
+drivers.folder=C:\\webdriver
```
Comment (shoggy) [inline src/test/resources/test.properties:2]: this makes build machine dependent

**Review by shoggy — APPROVED** [review]

**Pull request merged**